WebApp/autoreduce_webapp/reduction_viewer/utils.py

```diff
@@ -145,7 +145,7 @@ def createRetryRun(self, reductionRun, overwrite=None, script=None, variables=No
             import traceback
             logger.error(traceback.format_exc())
             logger.error(e)
-            new_job.delete()
+            #new_job.delete()
```

Comment:
Is this meant to be commented out?

Reply:
Yes, currently it's swallowing our stack traces. The most likely step to fail is the `save()` step, so attempting to delete the record will throw itself.
I'm planning on addressing this in #91 properly